### PR TITLE
Abehjati/eth-update-interface

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: ethereum/test/pyth.js [hunk 1]
```diff
@@ -251,6 +258,30 @@ contract("Pyth", function () {
         return replaced;
     }
 
+    function generateRawUknownBatchAttestation(
```
Comment (jayantk) [inline ethereum/test/pyth.js:261]: typo: Unknown

File: ethereum/scripts/createLocalnetGovernanceVaa.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        emitterAddress,
+        sequence,
+        dataHex,
+        [testSigner1PK],
+        0,
+        0
+    );
+}
+
+module.exports = function createGovernanceVaa(dataBuffer, sequence) {
```
Comment (jayantk) [inline ethereum/scripts/createLocalnetGovernanceVaa.js:105]: suggest calling this createLocalnetGovernanceVAA to make it clear that this is only for testing and shouldn't be used in production

File: ethereum/migrations/prod/10_pyth_make_interface_simpler.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require('dotenv').config({ path: "../.env" });
```
Comment (jayantk) [inline ethereum/migrations/prod/10_pyth_make_interface_simpler.js:1]: what's the difference between the migrations in `prod` and `prod-receiver`? (you may have explained this to me already but i've forgotten)

Comment (jayantk) [reply ethereum/migrations/prod/10_pyth_make_interface_simpler.js:1]: nevermind. i read the README and it explains it, so 👍

**Review by jayantk — APPROVED** [review]
LGTM

**Review by tompntn — APPROVED** [review]

File: ethereum/Deploying.md [hunk 4]
```diff
@@ -23,11 +23,18 @@ npx apply-registry
 # If it is a new chain you are deploying to, create a new env file and commit it to the repo.
 rm -f .env; ln -s .env.prod.xyz .env && set -o allexport && source .env set && set +o allexport
 
-# Perform the migration
+# Perform the migration step by step using `--to` argument. Some steps require a governance execution to be successful.
 # You might need to repeat the steps because of busy RPCs.
 # Also, sometimes the gases are not adjusted. Please update them with the network
 # explorer gas tracker.
-npx truffle migrate --network $MIGRATIONS_NETWORK
+npx truffle migrate --network $MIGRATIONS_NETWORK --to <step>
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply ethereum/Deploying.md:30]: Unfortunately it won't fail. Yeah I agree it's not good, let's get back to it later.

**Pull request merged**